projects/core/src/cms/cms.module.ts

```diff
@@ -5,11 +5,29 @@
  */
 
 import { ModuleWithProviders, NgModule } from '@angular/core';
-import { provideDefaultConfig } from '../config/config-providers';
-import { defaultCmsModuleConfig } from './config/default-cms-config';
+import {
+  defaultCmsModuleConfig,
+  newCmsModuleConfig,
+} from './config/default-cms-config';
 import { CmsService } from './facade/cms.service';
 import { PageMetaModule } from './page/page-meta.module';
 import { CmsStoreModule } from './store/cms-store.module';
+import { provideDefaultConfig, provideDefaultConfigFactory } from '../config';
+import { isFeatureEnabled } from '../features-config';
+
+const config = {
+  features: {
+    newCmsEndpoint: false,
```

Comment:
I guess we don't need to define this config here, as by default it will be false, or undefined. And customers can enabled it from the app module perspective.

Reply:
I've refactor the whole thing